cli/block/service.py

```diff
@@ -0,0 +1,44 @@
+"""
+Provide implementation of the block.
+"""
+import asyncio
+
+from accessify import implements
+from aiohttp_json_rpc import RpcGenericServerDefinedError
+
+from cli.block.interfaces import BlockInterface
+
+loop = asyncio.get_event_loop()
+
+
+@implements(BlockInterface)
+class Block:
+    """
+    Implements block.
+    """
+
+    def __init__(self, service):
+        """
+        Constructor.
+
+        Arguments:
+            service: object to interact with Remme core API.
+        """
+        self.service = service
+
+    def get(self, identifier):
+        """
+        Get a block by its identifier.
+        """
+        try:
+            block = loop.run_until_complete(
+                self.service.blockchain_info.get_block_by_id(block_id=identifier),
+            )
+
+        except RpcGenericServerDefinedError as error:
+            return None, error.message
+
+        except Exception as error:
+            return None, str(error)
+
+        return block.get('data'), None
```

Comment:
Besides data in result no more useful information?

Reply:
@Alladin9393, `data` contains `batches`, `header` and `header_signature`. `result` container only `data`. So it is no needed to proxy intermediate key `data` for the user. Just all data's context into `result` key.